crypto/src/hash/traits.rs

```diff
@@ -13,7 +13,7 @@ const BLAKE_HASH_SIZE: usize = 32;
 /// This defines the prefix that is added to the data prior to it being hashed
 pub trait Hash
 where
-    Self: From<Box<[u8]>>,
+    Self: From<Box<[u8]>> + AsRef<[u8]>,
```

Comment:
Does this need an implementation of `AsRef` as a trait bound? It would be nice if we could drop this from here so that types can still implement `Hash` either way and just need to implement `AsRef<[u8]>` if they need converting to hex.

Reply:
removed